tests/end-to-end/external-solution-integration/pkg/testsuite/send_event.go

```diff
@@ -1,54 +0,0 @@
-package testsuite
-
-import (
-	"time"
-
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/internal/example_schema"
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
-)
-
-// SendEvent is a step which sends example event to the application gateway
-type SendEvent struct {
-	state   SendEventState
-	appName string
-	payload string
-}
-
-// SendEventState represents SendEvent dependencies
-type SendEventState interface {
-	GetEventSender() *testkit.EventSender
-}
-
-var _ step.Step = &SendEvent{}
-
-// NewSendEvent returns new SendEvent
-func NewSendEvent(appName, payload string, state SendEventState) *SendEvent {
-	return &SendEvent{state: state, appName: appName, payload: payload}
-}
-
-// Name returns name name of the step
-func (s *SendEvent) Name() string {
-	return "Send event"
-}
-
-// Run executes the step
-func (s *SendEvent) Run() error {
-	event := s.prepareEvent()
-	return s.state.GetEventSender().SendEvent(s.appName, event)
-}
-
-func (s *SendEvent) prepareEvent() *testkit.ExampleEvent {
```

Comment:
This is still a valid scenario.

Reply:
renamed the file to pkg/testsuite/send_event_to_compatibility_layer.go